seismostats/analysis/tests/test_estimate_mc.py

```diff
@@ -207,3 +236,11 @@ def test_estimate_mc_maxc(setup_magnitudes):
     mc = mc_max_curvature(setup_magnitudes, delta_m=0.1, correction_factor=0.2)
 
     assert_equal(1.3, mc)
+
+
+def test_estimate_mc_bvalue_stability(swiss_2023_magnitudes):
```

Comment:
This should be tested also for different binning and stability range :)

Reply:
Another missing test: one catalogue that fails to be stable enough.

Reply:
test for 0.1 binning added